components/dashboard/src/settings/ProfileInformation.tsx

```diff
@@ -0,0 +1,202 @@
+        const [errorMessage, setErrorMessage] = useState("");
+        const [visible, setVisible] = useState(shouldNudgeForUpdate(user!));
+
+        const saveProfileState = () => {
+            const error = ProfileState.validate(profileState);
+            setErrorMessage(error);
+            if (error) {
+                return;
+            }
+            const updatedUser = ProfileState.setProfileState(user!, profileState);
+            setUser(updatedUser);
+            getGitpodService().server.updateLoggedInUser(updatedUser);
+            setVisible(shouldNudgeForUpdate(updatedUser!));
+        };
+
+        const cancelProfileUpdate = () => {
+            setProfileState(original);
+            saveProfileState();
+        };
+
+        return (
+            <Modal
+                title="Update Your Profile Information"
+                visible={visible}
+                onClose={cancelProfileUpdate}
+                closeable={true}
+                className="_max-w-xl"
+                buttons={
+                    <div>
+                        <button className="secondary" onClick={cancelProfileUpdate}>
+                            Cancel
+                        </button>
+                        <button onClick={saveProfileState}>Save</button>
+                    </div>
+                }
+            >
+                <ProfileInformation
+                    profileState={profileState}
+                    errorMessage={errorMessage}
+                    updated={false}
+                    setProfileState={setProfileState}
+                />
+            </Modal>
+        );
+    }
+}
+
+export default function ProfileInformation(props: {
+    profileState: ProfileState.ProfileState;
+    setProfileState: (newState: ProfileState.ProfileState) => void;
+    errorMessage: string;
+    updated: boolean;
+    children?: React.ReactChild[] | React.ReactChild;
+}) {
+    return (
+        <div>
+            <h3>Profile</h3>
+            <p className="text-base text-gray-500 pb-4 max-w-2xl">
+                The following information will be used to set up Git configuration. You can override Git author name and
+                email per project by using the default environment variables <CodeText>GIT_AUTHOR_NAME</CodeText>,{" "}
+                <CodeText>GIT_COMMITTER_NAME</CodeText>, <CodeText>GIT_AUTHOR_EMAIL</CodeText> and{" "}
+                <CodeText>GIT_COMMITTER_EMAIL</CodeText>.
+            </p>
+            {props.errorMessage.length > 0 ? (
+                <div className="dark:bg-gray-800 bg-gitpod-kumquat-light rounded-md p-3 text-gitpod-red text-sm mb-2">
+                    {props.errorMessage}
+                </div>
+            ) : null}
+            {props.updated ? (
+                <div className="dark:bg-gray-800 bg-gray-300 rounded-md p-3 text-gray-700 dark:text-gray-200 text-sm mb-2">
+                    Profile information has been updated.
+                </div>
```

Comment:
**suggestion**: Ideally, this can safely use the new **[alert component](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/src/components/Alert.tsx)** while :a: using the message variant and :b: being closable. However, we can leave this out of the scope of this PR for now and create a follow-up issue.